### PR TITLE
Make the stock broker implementation functional

### PR DESCRIPTION
**Pull request by pmorie**
This PR:

- finishes implementing a minimally usable APISurface
- adds an example implementation that is similar to the ups-broker from service-catalog

### CODE REVIEW

File: examples/service-instance.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: skeleton-service-instance
+  namespace: test-ns
+spec:
+  clusterServiceClassExternalName: skeleton-example-service
+  clusterServicePlanExternalName: default
+  parameters:
+    param-1: value-1
```
Comment (LiliC) [inline examples/service-instance.yaml]: Maybe we could make this less vague, would be helpful to newcomers? Or add a comment above it?

File: examples/service-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
```
Comment (LiliC) [inline examples/service-binding.yaml]: For these examples would be good to include something about them in the README maybe?

Comment (LiliC) [reply examples/service-binding.yaml]: Also considering this is an example repo anyways, maybe we could rename this directory from `examples` to `manifests` or something better? :) WDYT?

File: examples/service-instance.yaml (hunk 1 above)
Comment (pmorie) [reply examples/service-instance.yaml]: Agree, this was basically a lift-n-shift from service-catalog

File: examples/service-binding.yaml (hunk 2 above)
Comment (pmorie) [reply examples/service-binding.yaml]: I had the same thought to do.  I'll merge this and do it in a follow-up.

**Pull request merged**